docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+needs to train on data that contains the ground truth, known as the 
+`dependent_variable`. In this example, the ground truth is available in each 
+document as the actual value of `FlightDelay`.
+
+If your source data consists of some documents that contain a 
+`dependent_variable` and some that do not, the model is trained on the 
+`training_percent` of the documents that contain ground truth. However, 
+predictions are made against all of the data. The current implementation of 
+{classanalysis} supports a single batch analysis for both training and 
+predictions.
+
+
+. Create a {dfanalytics-job}.
++
+--
+Use the {ref}/put-dfanalytics.html[create {dfanalytics-jobs}] API as you can see 
+in the following example:
+
+[source,console]
+--------------------------------------------------
+PUT _ml/data_frame/analytics/model-delayed-flights
+{
+  "source": {
+    "index": [
+      "kibana_sample_data_flights"  <1>
+    ]
+  },
+  "dest": {
+    "index": "df-flight-delayed",  <2>
+    "results_field": "ml" 
+  },
+  "analysis": {
+    "classification": {
+      "dependent_variable": "FlightDelay",  <3>
+      "training_percent": 10  <4>
+    }
+  },
+  "analyzed_fields": {
+    "includes": [],
+    "excludes": [    <5>
+      "Cancelled",
+      "FlightDelayMin",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit": "100mb" <6>
+}
+--------------------------------------------------
+// TEST[skip:setup kibana sample data]
+
+<1> The source index to analyze.
+<2> The index that will contain the results of the analysis; it will consist of 
+a copy of the source index data where each document is annotated with the 
+results.
+<3> Specifies the variable we want to predict with the {classanalysis}.
+<4> Specifies the proportion of data that is used for training the model. In 
+this example we use 10% of the source data. The training data is randomly 
+selected and provide predictions from the remaining 90%. These percentages are 
+approximate. For most of the datasets, even a small proportion of the training 
+data (2-5%) results in good predictions.
+<5> Specifies fields to be excluded from the analysis. It is recommended to 
+exclude fields that either contain erroneous data or describe the 
+`dependent_variable`.
+<6> Specifies a memory limit for the job. If the job requires more than this 
+amount of memory, it fails to start. This makes it possible to prevent job 
+execution if the available memory on the node is limited.
+
+
+The API returns the following response:
+
+[source,console-result]
+--------------------------------------------------  
+{
+  "id" : "model-flight-delays",
+  "source" : {
+    "index" : [
+      "kibana_sample_data_flights"
+    ],
+    "query" : {
+      "match_all" : { }
+    }
+  },
+  "dest" : {
+    "index" : "df-flight-delayed",
+    "results_field" : "ml"
+  },
+  "analysis" : {
+    "regression" : {
+      "dependent_variable" : "FlightDelayMin",
+      "prediction_field_name" : "FlightDelayMin_prediction",
```

Comment:
I think this prediction_field_name is the only interesting output field.  Should we draw attention to it? If not, maybe this output is unnecessary and can be omitted.